pkgs/development/tools/misc/cc-tool/default.nix

```diff
@@ -0,0 +1,40 @@
+{ stdenv, fetchFromGitHub,
+  autoconf, automake, boost, libusb1, libtool, pkg-config
+}:
```

Comment:
This can probably be formatted a bit more nicely. I am usually lazy and let `nixpkgs-fmt` do the work ;).